redis/src/main/scala/com/avsystem/commons/redis/actor/RedisConnectionActor.scala

```diff
@@ -92,6 +92,18 @@ final class RedisConnectionActor(address: NodeAddress, config: ConnectionConfig)
     }
   }
 
+  /**
+    * Without separate "reconnecting" state we would not be able to drop old messages
+    * (especially Connected message if arrived after ConnectionClosed because of some race conditions).
+    * Without this state, we would process old ones instead of dropping them and creating a new connection.
+    */
+  private def reconnecting(retryStrategy: RetryStrategy): Receive = {
```

Comment:
I guess this behaviour should also react to other steering messages like Open, IncomingPacks, Release or Close

Reply:
My intention was to do not react to those as if we reconnecting, it means we already closed/lost connection so all old messages can be dropped. Handling those messages would cause reacting to messages from connection that does not longer exist